pkg/autherrors/autherrors.go

```diff
@@ -34,10 +34,9 @@ func NewInvalidTokenError() *AuthError {
 	}
 }
 
-func NewUnknownError(err error) *AuthError {
+func NewUnknownError() *AuthError {
```

Comment:
Not related to the current PR but addresses an issue left over from a previous PR.